specs/models/recipe.spec.js

```diff
@@ -0,0 +1,55 @@
+var specHelper = require('../spec_helper');
+var Recipe = require('../../models').Recipe;
+
+describe('Recipe Model test', () => {
+  beforeAll(() => {
+    specHelper.tearDown()
+    specHelper.testSetup()
+  });
+
+  test('It should exist', () => {
+    return Recipe.create({
+        name: 'test recipe 1',
+        url: 'www.test.com',
+        yield: 4.0,
+        calories: 1000,
+        totalTime: 300.5
+      })
+      .then(recipe => {
+        expect(recipe).toBeInstanceOf(Recipe)
+      })
+  })
+
+  test('It has attributes', () => {
+    return Recipe.create({
+        name: 'test recipe 2',
+        url: 'www.test.com',
+        yield: 4.0,
+        calories: 1000,
+        totalTime: 300.5
+      })
+      .then(recipe => {
+        expect(recipe.name).toBe('test recipe 2')
+        expect(recipe.url).toBe('www.test.com')
+        expect(recipe.yield).toBe("4")
```

Comment:
This datatype came back different than we expected. We expected 4.0 as a decimal.